src/Utilities.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Scanner;
+
+public class Utilities {
+
+    public static String[] collectPersonDetails() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("First name: ");
+        String name = scanner.nextLine();
+        System.out.print("Last name: ");
+        String lastName = scanner.nextLine();
+        System.out.print("Age: ");
+        String age = scanner.nextLine();
+        System.out.print("PESEL: ");
+        String pesel = scanner.nextLine();
+        scanner.close();
+
+        return new String[]{name, lastName, age, pesel};
+    }
+
+    public static Person registerPerson(String[] personDetails) {
+        Person person = null;
+        try {
+            return new Person(personDetails[0], personDetails[1], Integer.parseInt(personDetails[2]), Long.parseLong(personDetails[3]));
+        } catch (RuntimeException e) {
+            e.printStackTrace();
+            return null;
```

Comment:
zwracanie nulla jako wynik metody  nie jest najlepszym rozwiazaniem (wiem ze `BankAccount` pozniej sprawdzi czy person nie jest nullem) Lepiej uzyc ktores ztych metod:
* zwrocic `Optional<Person> ` (o tym beda pozniejsze zajecia)
* uzyc NULL OBJECT pattern
* rzucic wyjatek (albo poprostu nie lapac go wtym miejscu)

Reply:
Na chwile obecna nie wiem jak tego użyć,m wiec dodałem do Utilities walidacje czy można stworzyć taki obiekt Person który póżniej sprawdzałbym tak:
if (Utilities.validatePerson(details))
            Utilities.registerPerson(details);

Reply:
to na zajeciach poruszymy ten temat